Platforms/QemuQ35Pkg/QemuQ35Pkg.dsc

```diff
@@ -1035,6 +1037,11 @@ QemuQ35Pkg/Library/ResetSystemLib/StandaloneMmResetSystemLib.inf
     <LibraryClasses>
       NULL|MdeModulePkg/Library/LzmaCustomDecompressLib/LzmaCustomDecompressLib.inf
       DevicePathLib|MdePkg/Library/UefiDevicePathLib/UefiDevicePathLib.inf
+    <PcdsFixedAtBuild>
+      ## Forcibly enables the debugger with a 30 second initial breakpoint. This can be set from the cmdline by passing

```

Comment:
Force enabled will enabled a inifinite timeout.

Reply:
Yep, I see that now, misread the hob def.